app-editors/vim/vim-8.0.1230.recipe

```diff
@@ -138,7 +138,7 @@ INSTALL()
 	echo "set nocompatible" > $dataDir/vim/vimrc
 
 	# Workaround to get the themes and plugins work
-	echo "set runtimepath+=`finddir B_USER_SETTINGS_DIRECTORY`/vim" > $dataDir/vim/vimrc
+	execute "set runtimepath+=" . substitute(system("finddir B_USER_SETTINGS_DIRECTORY"), '\n\+$', '', '') . "/vim"
```

Comment:
What I wrote in https://github.com/haikuports/haikuports/issues/3088#issuecomment-423039780 was `vimscript`, not something for bash...

```diff
-	# Make arrow keys, backspace, delete, etc behave as expected.
-	echo "set nocompatible" > $dataDir/vim/vimrc
-	# Workaround to get the themes and plugins work
-	execute "set runtimepath+=" . substitute(system("finddir B_USER_SETTINGS_DIRECTORY"), '\n\+$', '', '') . "/vim"
+	cat << EOF > $dataDir/vim/vimrc
+" Workaround to get the themes and plugins work
+execute "set runtimepath+=" . substitute(system("finddir B_USER_SETTINGS_DIRECTORY"), '\n\+$', '', '') . "/vim"
+EOF
```

This diff should also fix #3097